src/components/layouts/pages/inspect/tree/Tree.js

```diff
@@ -117,20 +121,26 @@ function RuleList({ ruleSummaries, expandedRule, selectedCheck, onRuleClick, onC
                         </IconButton>
                     </ListItemSecondaryAction>
                 </ListItem>
-                {rule.checks.length && (
+                {checks.length ? (
                     <Collapse in={expandedRule === index} timeout="auto" unmountOnExit>
                         <List component="div" disablePadding>
-                            <CheckList checks={rule.checks} selectedCheck={selectedCheck} onCheckClick={onCheckClick} />
+                            <CheckList
+                                checks={checks}
+                                selectedCheck={selectedCheck}
+                                onCheckClick={onCheckClick}
+                                errorsMap={errorsMap}
+                            />
                         </List>
                     </Collapse>
-                )}
+                ) : null}
             </Fragment>
         );
     });
 }
 
-function CheckList({ checks, selectedCheck, onCheckClick }) {
-    return checks.map(({ context }, index) => {
+function CheckList({ checks, selectedCheck, onCheckClick, errorsMap }) {
+    let checksSorted = sortChecksByPage(checks, errorsMap);
+    return checksSorted.map(({ context }, index) => {
         return (
             <ListItem
```

Comment:
Clicking on Unrecognised context results in page scolling (in case when context is a path to `contentItem` without `mcid`).  So, `ListItem` with Unrecognised context should be disabled